ci/test_cpp.sh

```diff
@@ -38,6 +37,8 @@ pushd "${RAPIDS_DATASET_ROOT_DIR}"
 ./get_test_data.sh
 popd
 
+EXITCODE=0
+trap "EXITCODE=1" ERR
```

Comment:
Can you rearrange these lines a bit:

- Consolidate the `EXITCODE` lines
- Move the `set +e` below the `EXITCODE` lines for consistency with the other repositories
- Remove the `TODO` a few lines down
- Put the `rapids-logger` statement right before the subsequent `for` loop

Reply:
done, thanks!